data/individualNames.json

```diff
@@ -1,13 +1,11 @@
-//Individuals Names
-
-[ "Marie",
-  "Craig",
-  "Justin",
-  "Keith",
-  "Walter",
-  "Alex",
-  "David",
-  "Doug",
-  "Govinda",
-  "Larry"
+[ "Marie Schmidt",
+  "Craig Martin",
+  "Justin Tomlinson",
+  "Keith Nygaard",
+  "Walter Schlosser",
+  "Alex Uribe",
+  "David Zuluaga",
```

Comment:
I merged the PR but something to think about....

If the app ever wanted to show only the first name then it would be sort of tricky.   You would have to split on spaces (and handle if there were 3 names, or only one).   Instead of having a stronger data model with "firstName" and "lastName".  Not sure that I am married to the idea of separating but wanted to call out the issue with not separating.